log/sockstatlog/logger.go

```diff
@@ -50,10 +57,16 @@ type event struct {
 	Stats map[sockstats.Label]deltaStat `json:"s"`
 }
 
+// sockstatLogID reproducibly derives a new logid.PrivateID to be used for sockstat logging.
```

Comment:
I would place the caveats about spoofing somewhere and why we're not worried about it. Perhaps here?